Automattic/wp-calypso
# Make the Help Center app

This creates a Jetpack ready Help Center app.

### Bundles

This makes four ready-to-be-enqueued bundle of the Help Center to be used by Jetpack.

1. **Gutenberg**
    - Jetpack connected - renders the Help Center
    - Jetpack disconnected - renders just a button that links to /help.
2. **wp-admin**
    - Jetpack connected
    - Jetpack disconnected- renders just a button that links to /help.

### Translations

1. This pulls Calypso strings and creates Help Center translations files off them.
2. It then uploads these translation files to widgets.wp.com.
3.  Jetpack in turn download these files and enqueues them as translations.

### Change to ETK

This changes ETK to only the Help Center when it's not already loaded. In a future change, I'll remove all the Help Center code from ETK.

### Testing steps
It's not needed to test this as a product. But you can help by testing the build process.

1. cd `apps/help-center`.
2. run `yarn build`.
3. You should get an `dist` folder containing all the bundles mentioned above.
4. You should get a `dist/languages` folder containing JSON translations files.
